### ISSUE TITLE
How do I display the full suggestion detail directly

### ISSUE THREAD

**Issue by cuiyueshuai** [microsoft/monaco-editor#4426]
I want to display the full suggestion detail directly, rather than clicking the arrow extension to display it, something like that
![image](https://github.com/microsoft/monaco-editor/assets/13741688/1b0bc6f5-ae49-4544-ab8c-5201c0d5e647)

I look up to the interface Monaco/Editor/API/Editor/ISuggestOptions/showInlineDetails, but this is not what I want, how should I do

Thank you very much

**Comment by XueMeijing** [comment]
Do you mean show detail like this? If so, you can set the detail property in suggestions items
```javascript
monacoRef.current.languages.registerCompletionItemProvider('sql', {
  triggerCharacters: ['.'],
  provideCompletionItems: (model, position) => {
    return {
      suggestions: sqlKeywords.map((key) => ({
        label: key,
        kind: monaco.languages.CompletionItemKind.Keyword,
        insertText: key,
        detail: 'mysql',
      })),
    };
  },
});
```
![image](https://github.com/microsoft/monaco-editor/assets/35559153/0ff12e3d-ba5b-4712-b4c4-d3dd6d78b973)

but i don't know how to show ALL the items detail at the same time like vscode
![image](https://github.com/microsoft/monaco-editor/assets/35559153/2a87ba05-2e8a-4750-afc5-2942c5090389)

**Comment by rcjsuen** [comment]
For me, if I hit <kbd>Ctrl+Space</kbd> again it will display the second popup. Perhaps you can try that.

**Comment by cuiyueshuai** [comment]
@rcjsuen  Yes,  I hit Ctrl+Space display the second popup by pressing Ctrl+Space in the English input environment, but I want to display it directly without operating the display,  How can the code be configured? do you have a way ? thank you

> For me, if I hit Ctrl+Space again it will display the second popup. Perhaps you can try that.

**Comment by hgrourou** [comment]
+1
